tap_s3_csv/transform.py

```diff
@@ -169,10 +175,6 @@ def transform_recur(self, data, schema, path, source_type=None):
         if not isinstance(types, list):
```

Comment:
at this point, types is expected to be lists from transform_schema_recur. can remove this check